src/components/pages/CaseTable/CaseTable.js

```diff
@@ -106,11 +110,22 @@ const useStyles = makeStyles(theme => ({
       transform: 'scale(1.4)',
     },
   },
+  toolbar_options: {
+    borderRadius: '6px',
+    padding: '0.5rem',
+    display: 'flex',
+    alignItems: 'center',
+  },
+
   toolbar: {
+    padding: '1rem',
     display: 'flex',
     flexDirection: 'row',
-    margin: '5px',
-    color: 'darkblue',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: '6px',
+    width: '100%',
+    color: 'black',
```

Comment:
Color should be #215589

Reply:
I have changed the toolbar buttons to all be the same color.